README.md

```diff
@@ -1,34 +1,57 @@
 # D3B:Loss of Heterozygosity (LOH)
 
-LOH CWL workflow assesses the loss of heterozygosity(LOH) in the tumor for rare germline variants (gnomad_3_1_1_AF_popmax < 0.01).
+This CWL workflow assesses the loss of heterozygosity(LOH) in the tumor for rare germline variants (gnomad_3_1_1_AF_popmax < 0.01). This workflow has been deployed on [cavatica](https://cavatica.sbgenomics.com/u/d3b-bixu/tumor-loh-dev/apps/Loss_of_Heterozygosity) with a [proband only run](https://cavatica.sbgenomics.com/u/d3b-bixu/tumor-loh-dev/tasks/522d128a-2195-4c9c-8339-1709da16821d/) and a [trio run](https://cavatica.sbgenomics.com/u/d3b-bixu/tumor-loh-dev/tasks/d7f6b667-35ef-46a7-a666-970a78ef3175/)
```

Comment:
So to everyone who isn't a member of this project, these links are pretty useless.

Reply:
@dmiller15 I see, so what's the best solution to this ?

Reply:
A public app?

Reply:
A ticket is up for a public app, meanwhile can I go forward without changing these lines?

Reply:
Wouldn't you want to hold off merging this until you have the public app link so you can point people to that?

Reply:
I am thinking to make a new PR for public release

Reply:
Ok.